README.md

```diff
@@ -1,10 +1,68 @@
-Start with:
-`./treehouse-builder --chroot`
+# treehouse-builder
+
+Treehouse-builder is based on [Raspian](https://www.raspbian.org/) and allows the user to develop and tailor their own custom RPi images. The script will modify the latest Raspian image by installing packages, purging packages and executing custom commands, and then finally creates a bootable .img file that can be burned to the microSD card.
+
+## Instructions
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+```
+System requirements
+class10 microSD card (minimum 8 Gb)
+
+Required packages
+kpartx wget gpg parted qemu-arm-static
+
+To install the required packages, run the following command in Debian/Ubuntu: 
+sudo apt-get install kpartx wget gpg parted qemu-arm-static. 
+```
+### Getting Started 
```

Comment:
@mappuji Thank you for pointing out the corrections and improvements. I have made changes accordingly. For this one, I have added a link to ole--vagrant-treehouses, which will have instructions for the same.